dataland-frontend/tests/e2e/fixtures/lksg/LksgDataFixtures.ts

```diff
@@ -0,0 +1,174 @@
+import { faker } from "@faker-js/faker";
+import { LksgData, ProductionSite } from "../../../../build/clients/backend";
+
+import { randomYesNoUndefined } from "../common/YesNoFixtures";
+
+export function generateProductionSite(): ProductionSite {
+  const fakeSiteName = faker.company.name();
+  const yesNo = randomYesNoUndefined();
+  const fullFormattedFakeAddress =
+    faker.address.street() +
+    " " +
+    faker.address.buildingNumber() +
+    ", " +
+    faker.address.zipCode() +
+    " " +
+    faker.address.city() +
+    ", " +
+    faker.address.country();
+  const fakeGoodsAndServices = Array.from(
+    { length: faker.datatype.number({ min: 1, max: 5 }) },
+    faker.commerce.productName
+  );
+
+  return {
+    name: fakeSiteName,
+    isInHouseProductionOrIsContractProcessing: yesNo,
+    address: fullFormattedFakeAddress,
+    listOfGoodsAndServices: fakeGoodsAndServices,
+  };
+}
+
+export function generateArrayOfProductionSites(): ProductionSite[] {
+  return Array.from({ length: faker.datatype.number({ min: 1, max: 50 }) }, generateProductionSite);
```

Comment:
50 seems to be a bit much from my point of view. Also I would include 0 since that might trigger corner cases. So maybe between 0 and 5?

Reply:
Repaired.